Error with latest master : Unknown applies-on argument: OFF

I am trying to integrate this with NEURON as:

```
 → git diff --cached
diff --git a/.gitmodules b/.gitmodules
index bc0480f8..c27ea5b5 100644
--- a/.gitmodules
+++ b/.gitmodules
@@ -4,3 +4,6 @@
 [submodule "external/coreneuron"]
        path = external/coreneuron
        url = https://github.com/BlueBrain/CoreNeuron.git
+[submodule "external/coding-conventions"]
+       path = external/coding-conventions
+       url = https://github.com/BlueBrain/hpc-coding-conventions.git
diff --git a/CMakeLists.txt b/CMakeLists.txt
index cc59a290..2477b546 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -204,6 +204,11 @@ include(PythonDynamicHelper)
 add_subdirectory(src/nrniv)
 add_subdirectory(bin)

+# =============================================================================
+# CMake format and Clang format
+# =============================================================================
+add_subdirectory(external/coding-conventions/cpp)
+
 if (NRN_ENABLE_PYTHON)
   add_subdirectory(src/nrnpython)
 endif()
diff --git a/external/coding-conventions b/external/coding-conventions
new file mode 160000
index 00000000..159ee089
--- /dev/null
+++ b/external/coding-conventions
```

And trying to check `cmake-format` target gives:

```bash
→ make cmake-format
Unknown applies-on argument: OFF
Traceback (most recent call last):
  File "/Users/kumbhar/workarena/repos/bbp/nn/external/coding-conventions/cpp/cmake/bbp-cmake-format.py", line 144, in <module>
    sys.exit(0 if main() else 1)
  File "/Users/kumbhar/workarena/repos/bbp/nn/external/coding-conventions/cpp/cmake/bbp-cmake-format.py", line 135, in main
    args.source_dir, args.binary_dir, excludes_re, files_re
  File "/Users/kumbhar/workarena/repos/bbp/nn/external/coding-conventions/cpp/cmake/cpplib.py", line 133, in filter_git_modified
    modified_files = set(get_git_changes(cli_args.source_dir, cli_args.applies_on, git=cli_args.git_executable))
  File "/Users/kumbhar/workarena/repos/bbp/nn/external/coding-conventions/cpp/cmake/cpplib.py", line 125, in get_git_changes
    raise Exception(msg)
Exception: Unknown applies-on argument: OFF
make[3]: *** [external/coding-conventions/cpp/CMakeFiles/cmake-format] Error 1
make[2]: *** [external/coding-conventions/cpp/CMakeFiles/cmake-format.dir/all] Error 2
make[1]: *** [external/coding-conventions/cpp/CMakeFiles/cmake-format.dir/rule] Error 2
make: *** [cmake-format] Error 2
```

--- a/cpp/CMakeLists.txt
+++ b/cpp/CMakeLists.txt
@@ -182,9 +182,8 @@ endfunction(bbp_enable_static_analysis)
 
 bob_option(${PROJECT_NAME}_FORMATTING "Enable helpers to keep CMake and C++ code properly formatted"
            OFF)
-bob_option(${PROJECT_NAME}_FORMATTING_ON
-           "Specify changeset where formatting applies (see documentation for detailed information)"
-           all)
+bob_input(${PROJECT_NAME}_FORMATTING_ON "all" STRING
+          "Specify changeset where formatting applies (see documentation for detailed information)")
 bob_option(${PROJECT_NAME}_TEST_FORMATTING "Add CTest formatting test" OFF)
 bob_option(${PROJECT_NAME}_FORMATTING_NO_SUBMODULES "Exclude git submodules from formatting" ON)
 bob_option(${PROJECT_NAME}_CLANG_FORMAT "Enable helper to keep C++ code properly formatted" OFF)

